dataset-conversion/README.md

```diff
@@ -46,28 +46,27 @@ cd ..
 
 The best part about this model is that there is **no preprocessing required**! The model is directly trained on the raw data. The only data preparation step is to convert the data to the nnUNet format. The following commands are used for converting the dataset. 
 
-#### Reorientation
 
-For uniformity across the multi-site data, all images are converted to RPI orientation as a first step. This is done using [Spinal Cord Toolbox](https://spinalcordtoolbox.com) (SCT). From your unix-based terminal, run the following command from the root directory of the downloaded dataset:
+#### Conversion to nnUNet format
 
-```bash
-shopt -s globstar; for file in **/*.nii.gz;do sct_image -i ${file} -setorient RPI -o ${file}; done
-``` 
-This should do an in-place conversion of all the images (and labels) to RPI orientation.
+The next step is to convert the data to the nnUNet format using the `convert_bids_to_nnUNetv2_all_sci_data.py` script.
 
-#### Conversion to nnUNet format
+The script supports both region-based and multichannel labels. 
+
+Note: the script performs RPI reorientation of the images and labels; i.e., there is no need to reorient the images and 
+labels before running the script.
 
-The next step is to convert the data to the nnUNet format. Run the following command:
+Note: create venv and install `dataset-conversion/requirements.txt` before running the script.
 
 ```bash
 python convert_bids_to_nnUNetv2_all_sci_data.py --path-data ~/datasets/sci-zurich-rpi ~/datasets/sci-colorado-rpi
           --path-out ${nnUNet_raw} -dname tSCICombinedRegion -dnum 275 --split 0.8 0.2 --seed 50 --region-based
 ```
 
-This command takes as inputs the list of RPI-reoriented datasets, the output path to store the converted data, the dataset name, number, the ratio in which train and test subjects are split, the seed and the flag `--region-based` which used to create multiclass labels (SC and lesion).
+This command takes as inputs the list of datasets, the output path to store the converted data, the dataset name, the 
+dataset number, the ratio in which train and test subjects are split, the random seed, 
+and `--region-based` or `--multichannel` flag.
 
-> **Note**
-> This assumes the nnUNet has been successfully installed and the necessary nnUNet-related environment variables have been set. Please refer to the [nnUNet documentation](https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/installation_instructions.md) for more details.
 
 #### Cropping around the spinal cord
```

Comment:
Is this section still relevant?

Reply:
good catch, removed https://github.com/ivadomed/model_seg_sci/pull/87/commits/cc21fc814cb4e61052149b424766f4f9957eb26e